src/unix/bsd/netbsdlike/openbsd/mod.rs

```diff
@@ -540,6 +540,39 @@ impl siginfo_t {
         self.si_addr
     }
 
+    pub unsafe fn si_code(&self) -> ::c_int {
+        self.si_code
+    }
+
+    pub unsafe fn si_errno(&self) -> ::c_int {
+        self.si_errno
+    }
+
+    pub unsafe fn si_pid(&self) -> ::pid_t {
+        #[repr(C)]
+        struct siginfo_timer {
+            _si_signo: ::c_int,
+            _si_errno: ::c_int,
+            _si_code: ::c_int,
+            __pad1: ::c_int,
```

Comment:
the `_pad` type size is wrong.

it is defined as `int _pad[SI_PAD]` (an array of `::c_int`), and the number of elements is:
- `#define SI_PAD ((SI_MAXSZ / sizeof (int)) - 3)`
- `#define SI_MAXSZ 128`

Reply:
in fact it s just.a wrong copy paste from netbsd..